### PR TITLE
E168. Adding test cases for action allowed - answer_tags_controller 

### PR DESCRIPTION
**Pull request by snparab1704**
Please set the title of your pull request in the following format: `[Project id/Independent study/Issue fix/...]. project name`, eg., `E904. Integrating courses and assignments on Expertiza with Moodle`

We suggest you including the below information in your pull request:
- A description of the changes proposed in the pull request.
- @mentions of the person or team responsible for reviewing proposed changes.
---
About Expertiza Bot
- If you have any questions about comments given by the expertiza-bot [(example)](https://github.com/expertiza/expertiza/pull/1877#issuecomment-762412918), you could create a comment in your pull request with the format `/dispute [UUID1] [UUID2]` [(example)](https://github.com/expertiza/expertiza/pull/1877#issuecomment-762430057) then the professor and TAs will be notified.
- [Here is a video](https://tinyurl.com/internet-bots) about how to communicate to expertiza-bot.

### CODE REVIEW

File: spec/controllers/answer_tags_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,44 @@
+describe AnswerTagsController do
+
+  #factory objects required for "action_allowed" test cases
+  let(:instructor) { build(:instructor, id: 1) }
```
Comment (akk5597) [inline spec/controllers/answer_tags_controller_spec.rb:4]: Is this line needed? We are not using `instructor` anywhere else in the code.

**Review by akk5597 — COMMENTED** [review]
Other than that one line, lgtm, we can merge as is too.

**Review by akk5597 — APPROVED** [review]
Approving for merging just in case.

File: spec/controllers/answer_tags_controller_spec.rb (hunk 1 above)
Comment (snparab1704) [reply spec/controllers/answer_tags_controller_spec.rb:4]: Yeah actually not needed will remove this in next iteration.

**Pull request merged**